gluten-core/src/main/scala/org/apache/spark/rpc/GlutenExecutorEndpoint.scala

```diff
@@ -33,16 +33,16 @@ class GlutenExecutorEndpoint(val executorId: String, val conf: SparkConf)
   with Logging {
   override val rpcEnv: RpcEnv = SparkEnv.get.rpcEnv
 
-  val executorEndpoint: RpcEndpointRef =
-    rpcEnv.setupEndpoint(GlutenRpcConstants.GLUTEN_EXECUTOR_ENDPOINT_NAME, this)
-
   private val driverHost = conf.get(config.DRIVER_HOST_ADDRESS.key, "localhost")
   private val driverPort = conf.getInt(config.DRIVER_PORT.key, 7077)
   private val driverUrl =
     s"spark://${GlutenRpcConstants.GLUTEN_DRIVER_ENDPOINT_NAME}@$driverHost:$driverPort"
 
   @volatile var driverEndpointRef: RpcEndpointRef = null
 
+  val executorEndpoint: RpcEndpointRef =
```

Comment:
Can we remove `val executorEndpoint: RpcEndpointRef =`? Looks useless. Thanks!

Reply:
thanks, removed